vspace/src/main/java/edu/asu/diging/vspace/core/model/impl/LocalizedText.java

```diff
@@ -0,0 +1,60 @@
+package edu.asu.diging.vspace.core.model.impl;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import org.hibernate.annotations.GenericGenerator;
+import org.hibernate.annotations.Parameter;
+import com.fasterxml.jackson.annotation.JsonBackReference;
+import edu.asu.diging.vspace.core.model.ILocalizedText;
+
+@Entity
+public class LocalizedText implements ILocalizedText {
+    
+    @Id
+    @GeneratedValue(generator = "localized_text_id_generator")
+    @GenericGenerator(name = "localized_text_id_generator", parameters = @Parameter(name = "prefix", value = "LOCTEXT"), strategy = "edu.asu.diging.vspace.core.data.IdGenerator")
+    private String id;
+    
+    @ManyToOne
+    @JoinColumn(name = "LOC_EXH_LANG")
+    @JsonBackReference()
+    private ExhibitionLanguage exhibitionLanguage;
```

Comment:
this feels wrong. doesn't that mean that in the serialized version localized text doesn't have a reference to the language? I can't really see a situation in which we want the localized text from the language  (rather the other way around).

Reply:
changed it